src/lib/containers/Login.tsx

```diff
@@ -131,8 +133,8 @@ class Login extends React.Component<Props, State> {
     event.preventDefault()
     const redirectUrl = this.props.googleRedirectUrl
       ? this.props.googleRedirectUrl
```

Comment:
What does googleRedirectUrl do? Would that need to change as well once we have different providers?

Reply:
This is the redirect URL that we send to the OAuth provider (only Google for now).  If it's registered to our OAuth client for that provider, then they will follow it on successful login (and give us the access code).
It was poorly named prop if it's not Google.  Renaming means that this prop needs to be updated when this version of SRC is pulled into Synapse.org (otherwise it will break Google SSO).  Opened [SWC-5849](https://sagebionetworks.jira.com/browse/SWC-5849) to track.  Thanks!